airbyte-cdk/python/airbyte_cdk/connector.py

```diff
@@ -47,10 +47,14 @@ def configure(self, config: Mapping[str, Any], temp_dir: str) -> TConfig:
         """
 
     @staticmethod
-    def read_config(config_path: str) -> TConfig:
+    def read_json_file(config_path: str) -> TConfig:
```

Comment:
I was wondering here: should we change the return type to `Mapping[str, Any]` since this method has become more generic than just reading configs? 

Also, the override in `SourceS3. read_json_file` still use `Mapping[str, Any]` so base on your answer, I'll re-align the typing for the return types

Reply:
Mapping is a more accurate for this method so it's probably okay to change this signature